README.rst

```diff
@@ -148,6 +148,20 @@ The recipe supports the following options:
 **flake8**
     If set to True, run Flake8 code analysis. Default is ``True``.
 
+    Flake8 now takes advantage of ``flake8`` plugin system to cover some of
+    previously manually written linters. We've plugged in following extensions:
+
+    - ``flake8-blind-except``: Check for non correct try-except statements
+    - ``flake8-coding``: Check for utf8 headers
```

Comment:
The only problem with this check is that on python scripts and old stuff like that you **can not** add the header, it's a minor issue maybe...